heareval/predictions/task_predictions.py

```diff
@@ -542,11 +591,31 @@ def get_events_for_all_files(
     # Create events for all the different files. Store all the events as a dictionary
     # with the same format as the ground truth from the luigi pipeline.
     # Ex) { slug -> [{"label" : "woof", "start": 0.0, "end": 2.32}, ...], ...}
-    event_dict = {}
-    for slug, timestamp_predictions in tqdm(event_files.items()):
-        event_dict[slug] = create_events_from_prediction(
-            timestamp_predictions, idx_to_label
-        )
+    event_dict: Dict[
+        Tuple[Tuple[str, Any], ...], Dict[str, List[Dict[str, Union[float, str]]]]
+    ] = {}
+    if postprocessing:
+        postprocess = postprocessing
+        event_dict[postprocess] = {}
+        for slug, timestamp_predictions in tqdm(event_files.items()):
+            event_dict[postprocess][slug] = create_events_from_prediction(
+                timestamp_predictions, idx_to_label, **dict(postprocess)
+            )
+    else:
+        for median_filter_ms in [0.0, 60.0, 150.0]:
+            for min_duration in [0.0, 60.0, 150.0, 250.0]:
```

Comment:
Can we not define these in the PARAM_GRID? Or, create a separate EVENT_PARAM_GRID that copies PARAM_GRID and updates it with these?

Reply:
Added